its/projects/IncrementalPRAnalysis/IncrementalPRAnalysis/File1.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace IncrementalPRAnalysis
+{
+    public class File1
```

Comment:
We can name them `Unchanged1`, `Unchanged2` and `WithChanges` to better clarify the purpose.